### DISCUSSION TITLE
Google Add-on for slides

### DISCUSSION THREAD

**Discussion by rhewy** [jgraph/drawio#4219] [Q&A]
Is the google docs add on for slides still a thing.

It has gone missing on my slides and I cannot find it in the market place.

Chrome also just updated so it could be related to that.

If it is gone, is there still a way to embed draw.io diagrams on a slide.

Thanks,

Rob

**Comment by davidjgraph** [comment]
See https://github.com/jgraph/drawio/issues/4212
